dabudimesh/network.py

```diff
@@ -0,0 +1,109 @@
+from router import Router
+from utils import create_server, create_connection, socket_address
+from message import Message
+from utils import run_event_loop
+from config import MSG_MAX_LEN
+import asyncio
+from threading import Thread
+
+
+# TODO: create a common system of commands or messages for shell, nic and routers
+class NetworkInterfaceController:
+    def __init__(self, output: list):
```

Comment:
I tried doing this through a stream, but it turned out to be easier to test what the class outputs through a list. In general, it would be nice to come up with a message system for interaction between the console and the controller. Then the idea comes to mind about creating a common system of interaction through messages or commands between the console and the controller and between controllers.

Reply:
Done.